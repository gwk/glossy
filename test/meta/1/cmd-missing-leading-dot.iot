{
  'desc': 'The user creates `script.py` and intends to run it but forgets the leading `./`.',
  'compile': ['touch script.py'],
  'code': 1,
}
