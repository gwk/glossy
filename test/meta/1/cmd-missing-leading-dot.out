test-meta/1/cmd-missing-leading-dot
test process launch failed; file exists but invocation is missing a leading `./`.
test command failed: `script.py`

================================================================


################################################################
TESTS FOUND: 1; SKIPPED: 0; FAILED: 1.
